### ISSUE TITLE
Docu enhancement

### ISSUE THREAD

**Issue by StephanStS** [DeanCording/node-red-contrib-config#5]
Hello Dean,
I'd like to extend the documentation of this wonderful node with a couple of sentences, graphics and examples. May I, is it of interest?
E.g. see https://github.com/lutzer/node-red-contrib-easing or https://flows.nodered.org/node/node-red-contrib-finite-statemachine.

**Comment by StephanStS** [comment]
Done with pull request https://github.com/DeanCording/node-red-contrib-config/pull/6.

**Comment by StephanStS** [comment]
Hello @DeanCording,
in my fork I added a typo and also changed the background color of the configuration node. This shall help to read the nodes function name a bit better, because the black function name has a low contrast to the original node background color.
(The same issue is present at your node-red-contrib-fan, node-red-contrib-fs-ops, node-red-contrib-match nodes.)

If this is o.k. for you, I would emit another pull request.
Within this request I also changed the version from 1.1.2 to 1.1.3.
